src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -395,6 +395,15 @@ export const SavedSearch = PropTypes.shape({
   type: PropTypes.oneOf([SEARCH_TYPES.ISSUE, SEARCH_TYPES.EVENTS]),
 });
 
+export const Incident = PropTypes.shape({
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  status: PropTypes.string.isRequired,
+  query: PropTypes.string,
+  projects: PropTypes.arrayOf(PropTypes.number),
```

Comment:
Is this something that's stored in the model?

Reply:
Since project list is on the mockups it probably needs to be

Reply:
Should this be ids or slugs?

Reply:
Wasn't sure about this, both are used in various places

Reply:
Just marked it an array for now, we can figure out what format the children take later